_posts/2020-10-12-encyclopedia-influence-gospel-human-culture.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: post
+title:  "Semina Verbi: an online encylopedia of the influence of the Gospel on human culture"
+date:   2020-10-12 15:56:00 +0200
+categories: [bible, evangelization, culture]
+author: JohnRDOrazio
+comments: true
+---
+
+== Open knowledge ==
```

Comment:
For this line, I believe it would need to be a heading, or bold; the == syntax doesn't do anything in Markdown.

Reply:
Oh yes! I mix Markdown syntax with Wiki syntax! Sorry about that, will fix